azkaban-db/src/main/sql/create.version_set.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE version_set
+  (
+     num  INT NOT NULL auto_increment,
+     md5  CHAR(32) NOT NULL,
+     json VARCHAR(4096) NOT NULL,
```

Comment:
Version set table is only used to identify if the same combination of versions were used earlier as well, and if yes, what was the assigned id to it. Hence, the only use case for this table to determine the id based on md5, and along with id, we may need what does id describes? (json string).

Reply:
Sometimes, it will help us to find bugs based on when version set is created. In case of faulty image version or something, we can filter by date and all. It will be useful to keep.

Reply:
This is pending.